### DISCUSSION TITLE
Guides, tutorials & documentation in Jupyter/Colab Notebooks for v4.0

### DISCUSSION THREAD

**Discussion by axantillon** [tensorflow/cleverhans#1195] [Ideas]
(Inaugurating the Discussions Board! 🎉)

Hey,
As soon as v4.0 is released this could be the place to coordinate the development of Jupyter/Colab notebooks that can serve as guides for Adv. ML concepts and as tutorials for the library in a bit of a more interactive way!

On standby to start collaborating as soon as the new v is released,

**Comment by nick-jia** [comment]
Hi @andantillon! Thank you for offering help and sorry for the late reply! I'd love to collaborate to make this happen. Let's discuss the format of tutorials for this new version and I'll ask for feedback from other maintainers. Thank you again!
